.github/workflows/tools.yml

```
@@ -123,26 +119,18 @@ jobs:
             label: dependencies
             run: |
               NEW_VERSION=$(gh api repos/libuv/libuv/releases/latest -q '.tag_name|ltrimstr("v")')
-              VERSION_H="./deps/uv/include/uv/version.h"
-              CURRENT_MAJOR_VERSION=$(grep "#define UV_VERSION_MAJOR" $VERSION_H | sed -n "s/^.*MAJOR \(.*\)/\1/p")
-              CURRENT_MINOR_VERSION=$(grep "#define UV_VERSION_MINOR" $VERSION_H | sed -n "s/^.*MINOR \(.*\)/\1/p")
-              CURRENT_PATCH_VERSION=$(grep "#define UV_VERSION_PATCH" $VERSION_H | sed -n "s/^.*PATCH \(.*\)/\1/p")
-              CURRENT_SUFFIX_VERSION=$(grep "#define UV_VERSION_SUFFIX" $VERSION_H | sed -n "s/^.*SUFFIX \"\(.*\)\"/\1/p")
-              SUFFIX_STRING=$([[ -z "$CURRENT_SUFFIX_VERSION" ]] && echo "" || echo "-$CURRENT_SUFFIX_VERSION")
-              CURRENT_VERSION="$CURRENT_MAJOR_VERSION.$CURRENT_MINOR_VERSION.$CURRENT_PATCH_VERSION$SUFFIX_STRING"
-              if [ "$NEW_VERSION" != "$CURRENT_VERSION" ]; then
+              ./tools/dep_updaters/update-libuv.sh "$NEW_VERSION"
+              if [ $? = 0 ]; then
                 echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
```

I think a non-zero exit code would crash the GHA by default, adding `|| true` should swallow the error.
```suggestion
              (./tools/dep_updaters/update-libuv.sh "$NEW_VERSION" && \
                echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV) || true
```
But doesn't the script return 0 when there's no update? See line 26.

---

Yeah, I forgot to update the libuv and simdutf ones.

---

@aduh95 I opt for the same check as eslint and postject as the script could return a zero code even if the deps are up to date.